src/isodate/duration.py

```diff
@@ -4,11 +4,15 @@
 The class Duration allows to define durations in years and months and can be
 used as limited replacement for timedelta objects.
 """
-from datetime import timedelta
-from decimal import Decimal, ROUND_FLOOR
+from datetime import datetime, timedelta
+from decimal import ROUND_FLOOR, Decimal
+from typing import Any, Dict, Optional, Tuple, Union
```

Comment:
With `from __future__ import annotations` you can use `tuple` and `dict` instead of `Tuple` and `Dict`.

Also:

* `Union[a, b]` -> `a | b`
* `Optional[a]` -> `a | None`

I think pyupgrade may do some of these automatically if you add `from __future__ import annotations` first.

Reply:
29ba55e